tlmanager/controllers/proofs.py

```diff
@@ -63,8 +62,19 @@ def get_global_tree_consistency_proof(subroot, sublength):
     if sublength > global_tree.length:
         return {'status': 'error', 'message': 'Subtree length out of range'}
     
-    proof = global_tree.prove_consistency(sublength, subroot)
+    try:
+        proof = global_tree.prove_consistency(sublength, subroot)
+    except:
+        return {'status': 'error', 'message': 'Provided subroot was never root'}
```

Comment:
Essa mensagem é meio estranha... Talvez seja melhor "consistency proof for provided subroot was not found?"